docs/competition/teams/role-descriptions/reception.md

```diff
@@ -30,27 +32,40 @@ If they still object, they should be given a red lanyard and told to wear them a
 
 ## Handling members of the public
 
-Throughout the competition, receptionists typically get a lot of questions by the public. Any member of the public is allowed into the building. You just need to inform them about photos and videos being recorded.
+Throughout the competition, receptionists typically get a lot of questions by the public. Any member of the public is allowed into the building.
 
 You can give them a booklet and direct them to the arena if they seem interested.
 
 ## Handling Team Arrivals
 
-When a team first arrives or leaves, your role at reception is to log this and confiscate items on entry.
+When a team first arrives or leaves, your role at reception is to log this, collect items from the team and inform them of the next steps.
+
+If competitors have arrived before their team supervisor, they should be directed to wait in the waiting area by reception until their team supervisor arrives.
 
 Offer each competitor a bag of swag as they enter.
 
 ### Team entry/exit recording
 
-You will have a sign-in/sign-out sheet for team supervisors to record that their teams have arrived, make sure they fill in the details and sign it.
+You will have been provided with access to the competition helpdesk system. One of the things this records is whether teams are present. When a team first arrives each day, please mark the team as arrived in the system. If a teams arrive and leave throughout the day, please update the system accordingly.
```

Comment:
Without the supervisor actively signing a sheet, I think it might be possible for a team to sneak in without a supervisor. We could do either:

1. Tell receptionists to make sure there's a supervisor in their team.
2. Add a sign in sheet for the sole purpose that we make sure teams only arrive with their supervisor.

Reply:
After discussion with everyone it seems we'll go with option 1.